composer.json

```diff
@@ -16,10 +16,10 @@
     },
     "license": "GPL-3.0+",
     "require": {
-        "php": ">=5.4.0",
-        "ext-bcmath": "*"
+        "php": ">=5.4.0"
     },
     "require-dev": {
+        "ext-bcmath":                               "*",
```

Comment:
PR is always very OK, but you cannot get ride of a dependency just because you do not use on particular validator.
I propose you a way to resolve your PR below, finding an alternative to `bcmod`, in order to be able to completly remove this dep.